This patch fixes a bug that only occurs when compiling with clang due to wrong
handling of longjmp().
Upstream: https://lists.nongnu.org/archive/html/qemu-devel/2013-10/msg00503.html
--- cpu-exec.c.orig	2013-10-10 00:30:30.000000000 -0400
+++ cpu-exec.c	2013-10-10 00:32:37.000000000 -0400
@@ -681,6 +681,10 @@
              * local variables as longjmp is marked 'noreturn'. */
             cpu = current_cpu;
             env = cpu->env_ptr;
+#if !(defined(CONFIG_USER_ONLY) && \
+     (defined(TARGET_M68K) || defined(TARGET_PPC) || defined(TARGET_S390X)))
+            cc = CPU_GET_CLASS(cpu);
+#endif
         }
     } /* for(;;) */
 
